Buffer.hpp

```diff
@@ -13,14 +13,16 @@
 #include <stdexcept>
 #include <vector>
 
+#include "MaeParserConfig.hpp"
+
 namespace schrodinger
 {
 
 /**
  * A simple data class to hold unchanging character buffer data. Copies are
  * reference counted.
  */
-class BufferData
+class EXPORT_MAEPARSER BufferData
```

Comment:
So... a general question that applies to much in this review is that I wouldn't expect all of these classes to need to be exported. Did you default to exporting everything? Or add in things that were needed for tests to pass? Or...?

Reply:
Just things needed to get tests to pass.